### DISCUSSION TITLE
1.17.1

### DISCUSSION THREAD

**Discussion by DerpRealXD** [FabricMC/fabric-api#1549] [Player support] [answered]
I want to use wurst 1.17.1 but this is all that comes out
Could not find required mod: wurst requires {minecraft @ [>=1.17.1-rc2]}
Creating Empty clause ?
i have downloaded the latest fabric from the cursforge link yet still cannot boot up wurst hacks. Why?

**Comment by haykam821** [comment, marked as answer]
We do not support mods that allow players to gain an unfair advantage.

**Reply by TheAwesome98-Real** [reply]
have you ever heard of an anarchy server?

**Comment by DerpRealXD** [comment]
But this is for my own personal use. Not to play on large servers.

**Comment by mechanikate** [comment]
Point still stands.

**Reply by TheAwesome98-Real** [reply]
no it doesn't
